windows/setup.ps1

```diff
@@ -81,6 +81,16 @@ $ips = @()
 Foreach ($ip in $netcfg.IPAddress) { If ($ip) { $ips += $ip } }
 Set-Attr $result.ansible_facts "ansible_ip_addresses" $ips
 
+$env = New-Object psobject
```

Comment:
I wouldn't use $env as a variable name. It could interfere with an attempt to retrieve a particular environment variable further down in the file.

The following will now not function as expected.
```powershell
$env:MY_ENV_VAR
```